src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-usersgroups-list/smb-usersgroups-list.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container *ngIf="usersGroups$ | async as usersGroups">
+  <cd-table
+    [data]="usersGroups"
+    columnMode="flex"
+    [columns]="columns"
+    selectionType="single"
+    [hasDetails]="true"
+    (setExpandedRow)="setExpandedRow($event)"
+    (fetchData)="loadUsersGroups()"
+>
+  <cd-smb-usersgroups-details
+    *cdTableDetail
+    [selection]="expandedRow"
+  ></cd-smb-usersgroups-details>
+</cd-table>
+</ng-container>
+
+<ng-template
+  #groupsNamesTpl
+  let-row="data.row"
+>
+  <span *ngFor="let group of row.values.groups; let isLast=last">
+    {{ group.name }}{{ isLast ? '' : ', ' }}
+  </span>
```

Comment:
for a better visualization, the badges or the carbon alternate, tags?
https://carbondesignsystem.com/components/tag/usage/

move to carbon tags is still something in progress though: https://github.com/ceph/ceph/pull/60361